### PR TITLE
Save messages received

### PR DESCRIPTION
**Pull request by christopher77**
**This pull request contains:**

- Modifies initSocket function to save the messages from a chat

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -1,87 +1,87 @@
-const $listChannel = document.getElementById('listChannel');
-const $channelHeader = document.getElementsByClassName('channel-header')[0];
-const $formNewMessage = document.getElementById('newMessage');
-const $listMessage = document.getElementById('listMessage');
+const $listChannel = document.getElementById("listChannel");
```
Comment (condef5) [inline public/js/main.js]: Hey, you could change your prettier configuration to single quotes.

Comment (test0n3) [reply public/js/main.js]: Single quotes and include parenthesis for single input anonymous functions.

File: public/js/main.js [hunk 2]
```diff
@@ -1,87 +1,87 @@
...
 const socket = new WebSocket(`ws://localhost:3000/connection`);
-const messages = [
+const messages = JSON.parse(localStorage.getItem("messages")) || [
   {
     id: 1558261595489,
-    user: 'diegoC',
-    content: 'Papito mi rey',
-    channel: 'general'
+    user: "diegoC",
+    content: "Papito mi rey",
```
Comment (byteconde) [inline public/js/main.js]: Good standardization

File: public/js/main.js (hunk 1 above)
Comment (christopher77) [reply public/js/main.js]: yes , I will do it

**Pull request merged**